packages/http-client/src/ceramic-http-client.ts

```diff
@@ -131,6 +132,17 @@ class CeramicClient implements CeramicApi {
     }
   }
 
+  async getChainId(): Promise<string> {
```

Comment:
I think we should change this to something like this: 
```suggestion
  async getSupportedChains(): Promise<Array<string>>
```
This way we don't need to introduce a breaking change when nodes can support multiple anchor services.

Reply:
Good idea!  New version up that does this @oed.  Also updated the CAS side to also return an array in https://github.com/ceramicnetwork/ceramic-anchor-service/pull/151